rpc_jobs/rpc_aio.yml

```diff
@@ -335,6 +343,11 @@
                   holland.holland()
                 }} else if (env.STAGES.contains("Leapfrog Upgrade")) {{
                   deploy.upgrade_leapfrog(environment_vars: environment_vars)
+                  maas.deploy()
+                  maas.verify()
+                  tempest.tempest()
+                  kibana.kibana(env.KIBANA_SELENIUM_BRANCH)
+                  holland.holland()
```

Comment:
Holland relies on the support role which is being [removed from RPC testing](https://github.com/rcbops/rpc-openstack/pull/2343), so we should probably not add any further holland testing.

Holland should be tested in the support role repo, but that is obviously a separate issue.